lib/java/com/google/android/material/lists/listdatatypes/ListListData.java

```diff
@@ -0,0 +1,22 @@
+package com.google.android.material.lists.listdatatypes;
+
+import java.util.List;
+
+/** Implementation of {@link CustomListData} for {@link List}s */
+public class ListListData<T> implements CustomListData<T> {
```

Comment:
`ListListData`? Doesn't seem very conventional - I suggest we should rename the class to something like `ListData`?

Reply:
Yes, I was trying to stick to the naming convention of (Data Type) + ListData. I renamed it to just ListData.